### DISCUSSION TITLE
Who wants to be hired (July 2023)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#52398] [Show and tell]
# What is this?
This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Resume/CV:
  Website:
  Email:

Good luck with interviewing!

---

If you got hired through this job board, please reach out to me (hassan@vercel.com) so I can send you some swag :)

If you're a company looking to post a role, use #52399.

We've had yet another [*success story*](https://twitter.com/nutlope/status/1610667757565083648) of a developer being hired!*

**Comment by jensen** [comment]
Location: Vancouver, BC
Remote: Yes
Willing to relocate: No
Technologies: Full-Stack JavaScript (Check GitHub)
Resume/CV: https://www.linkedin.com/in/karljensendev/
Website: https://github.com/jensen

Recently shipped: https://alicent.ai/

**I am only looking for contract work at this time.**

**Comment by Young-Einstein10** [comment]
Location: Lagos, Nigeria
Remote: Yes
Willing to relocate: Yes
Role: Frontend Developer
Technologies: HTML, CSS, Javascript, React, NextJS, Typescript, VueJS, EmberJS, GraphQL, Node, Express, Prisma, NextAuth
Resume/CV: https://drive.google.com/file/d/1cxJrxziEZExPi_aIwtFF80OLlVqIPa0N/view
Website: https://abdulyusuf.me
LinkedIn: https://www.linkedin.com/in/abdulrahman-yusuf-a44463176
Github: https://github.com/Young-Einstein10

Open to Full-Time / Part-Time / Contract

**Comment by eashish93** [comment]
Location : Delhi (NCR), India
Remote: Yes
Willing to relocate: No
Role: Full Stack Developer
Technologies: HTML, CSS, JavaScript, NextJS, Typescript, Node, Firebase, MongoDB, 
Website: https://hackprogramming.com, https://minform.io, https://solutionportal.net
Currently working: https://minform.io, Client project: taspin.com (project management for freelancers),
LinkedIn: https://www.linkedin.com/in/eashish93/
Github: https://github.com/eashish93

**Only looking for contract work - part time or 25-30hrs/week**

**Reply by creteurlouis** [reply]
Hi eashish93,
Please feel free to apply here : https://www.acoustic-wells.com/careers

**Comment by peeter-tomberg** [comment]
Location: Estonia
Remote: Yes
Willing to relocate: Yes
Technologies: React, TS, Terraform, Docker, SQL, and Document dbs. 
Resume/CV: https://www.linkedin.com/in/peetertomberg/
Website: https://fvst.dev
Email: peeter.tomberg@gmail.com

**Comment by jessepinho** [comment]
## Senior full-stack freelance engineer

Hey all! Jesse Pinho here 👋🏻

I’m a senior full-stack engineer and technical advisor with deep experience in both backend and frontend — from Rails and Node (NestJS/Express) on the backend to React, NextJS, and Gatsby on the frontend.

I have availability for the next few months for new projects. My sweet spot is working with companies who have a new web-based project to build from the ground up: building the app, writing unit tests, writing documentation for future maintainers, and conducting a handoff to ensure its ongoing success. I also enjoy joining an existing team to get project launches across the finish line on a solid technical foundation.

My resume is here: https://jessepinho.com/assets/resume.pdf and my email is [jesse@jessepinho.com](mailto:jesse@jessepinho.com)

Shoot me a message if you’d like to chat about working together! Looking forward to hearing from you 🙂

Location: Los Angeles, CA, USA
Remote: Yes
Willing to relocate: No
Technologies: React, NextJS, TypeScript, Jest, Gatsby, Webpack, NodeJS, NestJS, Ruby on Rails, React Native
Resume/CV: https://jessepinho.com/assets/resume.pdf
Website: https://jessepinho.com/
Email: [jesse@jessepinho.com](mailto:jesse@jessepinho.com)

**Comment by nemanjam** [comment]
Location: Serbia, GMT +2
Remote: Yes
Willing to relocate: Maybe
Technologies: React, Next.js, JavaScript/TypeScript, Node.js, TailwindCSS, Jest, Cypress, Prisma, PostrgreSQL, MongoDB, Docker, C#/.NET
Resume/CV: Available on request, please send me an email
GitHub: https://github.com/nemanjam
LinkedIn: https://www.linkedin.com/in/nemanja-mitic
Email: nemanja.mitic.elfak [at] hotmail.com


I am a full stack developer with a Master's degree in Computer Science and 4 years of professional experience with JavaScript and C#/.NET. In the last few years my main focus has been the React/Next.js ecosystem, followed by Node.js. I am a JavaScript professional and a Python beginner interested to get more exposure to it. I have also worked with ASP.NET in the past which is not my primary focus at the moment. I have a basic understanding of DevOps tools.

A quick and easy way to assess my current skill level, the quality of my code and documentation, and my preferred tech stack (Next.js, TypeScript, React Query, Prisma, Postgres, TailwindCSS, Docker, Github Actions) is to check out my latest project: https://github.com/nemanjam/nextjs-prisma-boilerplate .

I am looking for a full-time remote position. We can discuss relocation after some time spent in the company. You can check out my other projects at https://github.com/nemanjam . I have detailed resume available, you can tell me about your company and project by email: nemanja.mitic.elfak [at] hotmail.com or by LinkedIn: https://www.linkedin.com/in/nemanja-mitic .

**Comment by Avas77** [comment]
### Frontend Engineer

Hi all! My name is Avas Bajracharya. Nice to meet you

Location: Kathmandu, Nepal
Remote: Yes 
Willing to relocate: Yes
Technologies: HTML, CSS, Javascript, React.js, Next.js, Typescript, Graphql, Docker 
Resume/CV: https://drive.google.com/file/d/1D6v32bZ0kvwOo_Ywr8fzhLfs4nhFqH6j/view
Website: https://avas-portfolio.web.app/
Email: avasbajracharya9@gmail.com
Github: https://github.com/Avas77

**Comment by milovangudelj** [comment]
## Web developer / UI designer

Hi! I design and develop engaging websites and delightful digital experiences.

My name is Milovan, but please call me Milo. I'm a computer science student at the University of Padua and I'm looking for an exciting new opportunity.

You can find me either on Next.js' or Astro's official discord severs helping people and contributing to Astro's i18n efforts for their documentation (I'm translating it to Italian :it:).

---

Location: Italy
Remote: Yes
Willing to relocate: Maybe in the future
Technologies: Next.js, Sanity, Hygraph, Supabase, Prisma, TailwindCSS
Resume/CV: https://milovangudelj.com/portfolio
Website: https://milovangudelj.com/
Email: [milovan.gudelj@gmail.com](mailto:milovan.gudelj@gmail.com)

**Comment by coolidev** [comment]
## Full Stack Developer - Web/Blockchain

Location: USA
Remote: Yes
Willing to relocate: No
Technologies: JavaScript, NodeJS, Solidity
Resume/CV: https://www.linkedin.com/in/jgoodidea
Website: https://leroyone.netlify.app
Email: gloproman0211@gmail.com

**Comment by Apestein** [comment]
Location: Atlanta GA, USA
Remote: Remote/Hybrid only
Willing to relocate: Yes
Technologies: React, NextJS, Typescript, Tailwind. T3 stack. For complete stack, see website.
Resume/CV: https://www.linkedin.com/in/luc-ledo/
Website: https://lucledo.dev/
Email: [lucledo@protonmail.com](mailto:lucledo@protonmail.com)

Summary
Full-stack web developer with a cutting-edge tech stack and a talent for streamlining processes. I pride myself in being a well-rounded developer, able to work on all aspects of app development. My ideal position is working with React, NextJS, Typescript and Tailwind. Pursuing innovative companies where my passion for building cutting-edge applications can shine.

**Comment by arevalolance** [comment]
Location: Philippines (GMT+8)
Remote: Yes
Willing to relocate: No
Technologies: React, Next.js, JavaScript, TypeScript, HTML, CSS, TailwindCSS, Go, Java
Résumé/CV: https://read.cv/arevalolance
Website: https://arevalolance.com
Email: hi@arevalolance.com

---

More info:

    - Available for full-time remote work
    - View more of my projects here: https://arevalolance.com/projects or https://github.com/arevalolance
    - LinkedIn: https://linkedin.com/in/arevalolance

**Comment by jaocfilho** [comment]
Location: Rio de Janeiro, Brazil
Remote: Yes
Willing to relocate: No
Technologies: Typescript, NextJS, Django, Python, TailwindCSS, Postgres, Docker, Supabase, Turborepo and React Testing Library.
Resume/CV: [resume](https://1drv.ms/b/s!ApcrtYUkP7tZwndW7QV3QuFNk1f9?e=ASBGWO)
Website: [github](https://github.com/jaocfilho) | [linkedin](https://www.linkedin.com/in/jaocfilho/)
Email: augustoliveira95@outlook.com

I invite you to take a look at my work on my github profile. I have big open source projects on production that proves my work experience with React, NextJS, Typescript and many other bleeding edge technologies.

You can also look at my latest project, [IntellisyncAI](https://www.intellisyncai.com/), a open-source no-code AI SaaS that allow users to talk with their documents. You can see its source code [here](https://github.com/jaocfilho/odyssey)

**Comment by ghubnab99** [comment]
Location: Bahrain (Middle East)
Remote: Yes
Willing to relocate: No
Technologies: HTML, CSS, Javascript, React, NextJS, Typescript, VueJS, GraphQL, Node, Express, Prisma, NextAuth, Firebase, Supabase, Mongodb, SQL Server, PostgreSQL
Resume/CV: [Resume](https://drive.google.com/file/d/1wuVh1wGlSKwb1sjuJol_Y5I5JmnvczUz/view?usp=sharing)
Website: [Portfolio](https://portfolio-three-d-fp9y.vercel.app/)
Email: mnabil777@gmail.com

**Comment by onursagir** [comment]
Location: Rotterdam, the Netherlands
Remote: Yes (possibly hyrid)
Willing to relocate: No
Technologies: Typescript, React, React Native, NodeJs, Docker
Resume/CV:https://www.linkedin.com/in/onursagir/
Website: - 
Email: onuur.s@gmail.com

Currently open for freelance work

**Comment by klasikh** [comment]
Location: Cotonou, Benin
Remote: Yes
Willing to relocate: Yes
Technologies: HTML, CSS, JavaScript, PhP, TypeScript, VueJS, ReactJS, NextJS, NextAuth, NodeJS, GraphQL, MongoDB, MySQL
Resume/CV: [https://www.linkedin.com/in/angehk/](https://www.linkedin.com/in/angehk/)
Website:[https://github.com/klasikh]( https://github.com/klasikh)
Email: [angehermel@gmail.com](mailto:angehermel@gmail.com)

**Comment by sbalayan1** [comment]
Location: Honolulu, HI
Remote: Yes
Willing to relocate: No
Technologies: HTML, CSS, JavaScript, TypeScript, ReactJS, NextJS PostgreSQL, Ruby, Ruby on Rails, Python, Jest, React Testing Library
Resume/CV: https://www.linkedin.com/in/sean-balayan/
Website: https://sean-balayan-io.onrender.com/ | https://github.com/sbalayan1
Email: balayans2014@gmail.com

**Comment by brandonperfetti** [comment]
Location: Orange County, CA

Remote: Yes

Willing to relocate: No

Technologies: JSX, Tailwind CSS, JavaScript, Typescript, GraphQL, React, Gatsby.js, Next.js, Vite, Jest, Codeship, Github Action, Netlify, Vercel

Resume/CV: https://www.brandonperfetti.com/assets/bp-cv.pdf

Website: https://www.brandonperfetti.com/

Email: brandon@brandonperfetti.com

**Comment by edyCodeforLife** [comment]
Location: Indonesia (UTC +7)
Remote: Yes

Willing to relocate: No

Technologies: React, Next.js, vue js, nuxt js, JavaScript, TypeScript, HTML, CSS, React Query, Chakra-UI, Tailwind CSS, Framer Motion, GraphQL, Wagmi, Redux Toolkit, Web3.js, Web3React, Rainbow Kit, Web3Auth, Emotion, Styled Components, webpack, Twilio, ant design, react context, unit test.

Résumé/CV: contact me on linkedin or email me to get the resume

Email: lzoulzs@gmail.com

Linkedin: https://www.linkedin.com/in/edy-s-565718141/

---

Open to part-time or full-time work.

I'm a front-end developer with 6 years of experience. For now i am working as Senior frontend developer from Indonesia Big tech company (GoTo Group), and i have very good experience in financial services in many company, building the web apps from scratch and create architecture flow. For now, i want to look for remote job as senior frontend developer or lead frontend developer. Please reach me out if we can have a discussion about this. Thank you.

**Comment by maximusworks** [comment]
Location: Austria
Remote: Yes
Willing to relocate: No
Technologies: MEVN, Vue/Laravel, Vue/Django
Resume/CV: https://www.linkedin.com/in/derek63
Website: https://jaguarman.netlify.app/
Email: derekliman63@gmail.com

**Comment by merndev1101** [comment]
Location: Canada
Remote: Yes
Willing to relocate: No
Technologies: MERN, TypeScript, Next.js
Resume/CV: https://www.xing.com/profile/Alex_Hong020720/cv
Website: https://merndev.netlify.app/
Email: along.hexo@gmail.com

**Comment by jukkun** [comment]
Location: Uttarkashi
Remote: Yes
Willing to relocate: Yes
Technologies: React, TypeScript, NextJS (App Router), Astro, TailwindCSS
Resume/CV: [PL](https://peerlist.io/raju_) [GH](https://github.com/Kiyoponn)
Website: -
Email: rajukhattri@proton.me

**Comment by mmajudith** [comment]
Location: Nigeria
Remote: Yes
Willing to relocate: Yes
Technologies: JavaScript, Typescript, ReactJs, NextJs, Reduxtoolkit, TailwindCss, mongoDB, firebase...

Resume/CV: https://www.dropbox.com/s/v5n7dpn1v6zn3nk/JudithOnyekwelu.pdf?dl=0

Website: https://judith-portfolio.vercel.app/
Email: mmajudith2000@gmail.com

**Comment by byamasu-patrick** [comment]
Location: Malawi
Remote: Yes
Willing to relocate: Yes
Technologies: JavaScript, Typescript, ReactJs, NextJs, Redux, TailwindCSS, PostgreSQL, MongoDB, Supabase
Resume/CV: [https://drive.google.com/file/d/1CO3iTQx_OBdUzo_dsa_0Gz-eKPd1LANT/view?usp=sharing](url)
Website: [https://www.byamasupatrick.com/](url) | [https://www.linkedin.com/in/byamasu-patrick-paul-1343a61a9/](url)
Email: ptrckbyamasu@gmail.com

**Comment by codepool867** [comment]
Location: Fort Mohave, AZ
Remote: Yes
Willing to relocate: No
Technologies: ReactJS, NextJS, VueJS, AngularJS, EmberJS, Redux, Html, CSS, JavaScript, TypeScript, TailwindCSS, MUI, Flutter, Dart, Lit
Resume/CV: https://docs.google.com/document/d/17ZzIKgih9cL15L49L5j94n8fv6BDunXh/edit?usp=drive_link&ouid=101618368071183469675&rtpof=true&sd=true
Website: https://jeffrey-childers.me
Email: royalmediaus@gmail.com
LinkedIn: https://linkedin.com/in/jeffrey-childers-tech
OpenTo: FrontEnd Developer, FrontEnd Enginner, FullStack Engineer

**Comment by Kashan-Haider** [comment]
Location: Lahore
Remote: Yes
Willing to relocate: No
Technologies: Full Stack Web Developer in Next JS 13 
Website: https://portfolio-awanpro786-gmailcom.vercel.app/
Email: awanpro786@gmail.com

**Comment by geektalent1010** [comment]
Location: Singapore
Remote: Yes
Willing to relocate: No
Technologies: MERN, Next.js, Vue.js, Angular, Laravel, MySQL, Tailwind, 
Resume/CV: https://drive.google.com/file/d/1ylwb-V5XcjvTdzMvLblIfowozdzbrp_2/view?usp=drive_link
Website: https://kiet0201.web.app/
Email: geektalent1010@gmail.com

**Comment by devknight216** [comment]
### **TypeScript/React/Next.js/Node.js/GraphQL/AWS Developer**
Location: Ukraine
Remote: Yes
Willing to relocate: No
Technologies: JavaScript, TypeScript, React/Next.js, Vue, Svelte, NodeJS, Python, GraphQL, AWS, Docker, MySQL, PostgreSQL, MongoDB
Website: https://igorkurgan.vercel.app/
Email: igorkrugandev@gmail.com

**Comment by victorcodess** [comment]
Location: Lagos, Nigeria
Remote: Yes
Willing to relocate: No
Role: Frontend Developer
Technologies: HTML, CSS3/SCSS/SASS, Javascript, React, NextJS, Typescript, Redux, Git, NextAuth, Formik, Tailwind CSS, Material UI, Framer Motion, GSAP, Bootstrap.
Resume/CV: https://drive.google.com/file/d/1XZPlj1js43hdIOH63nF0V_n4x_p80vhw/view
Website: https://www.victorwilliams.me/
Email: victor.williams@stu.cu.edu.ng

**Open to Internship / Part-Time / Contract**

**Comment by hex-grammer** [comment]
Location: Makassar, Indonesia
Remote: Yes
Willing to relocate: No
Role: Full-Stack Developer
Technologies: HTML, CSS, Javascript, React, NextJS, Typescript, Prisma, NextAuth, Tailwind CSS, MySQL, PHP
Resume/CV: https://drive.google.com/file/d/1SpOMSEnHu2scee-8x918IYLmGvLD0vrF
Website: https://rizaltsx.com/about
LinkedIn: https://www.linkedin.com/in/rizal-iswandy/
Github: https://github.com/hex-grammer
Email: rizal@xolusi.com

Open to Full-Time / Part-Time / Contract

**Comment by Patag1** [comment]
Location: Santa Fe Capital, Argentina
Remote: Yes
Willing to relocate: Yes
Role: preferred Front-end Developer (full-stack capabilities)
Technologies: HTML, CSS, Javascript, Typescript, Tailwind, React, Zustand, NextJS, Prisma, PostgreSQL, whole list in LinkedIn
Resume/CV: https://drive.google.com/file/d/1zDgpJoBsCzycVgjgu6q53GBUTLj4xfiS/view?usp=sharing
Linktree: https://linktr.ee/agusasso

Internship (paid) // Freelance // Part/Full-time

**Comment by Stekac** [comment]
Hi, 
Im Stefan, 7+ years of professional experience.

Location: Skopje, North Macedonia
Remote: Yes
Willing to relocate: No
Technologies: Typescript, React, React Native, React Query, TRPC, NextJS, Prisma, Node, Express, AWS, etc..
Resume/CV: [My CV](https://docs.google.com/document/d/1rbpr7Oy7Qk8Cbq2bPWWCpXrE7l3MO_De/edit?usp=sharing&ouid=101303371366643581120&rtpof=true&sd=true)
Website: [My Linkedin](https://www.linkedin.com/in/stefan-dejanovski/)
Email: stefan.dejanovski@gmail.com

**Comment by FredySandoval** [comment]
![MacBook Pro 16_ - 1 (1)](https://github.com/vercel/next.js/assets/45242501/24c2fea8-bf2a-438f-9b34-e370bb574c8d)

<p align="center">
  <a target="_blank" href="https://youtu.be/sr2ndSHVX7w">

[![Portfolio Video](https://github-production-user-asset-6210df.s3.amazonaws.com/45242501/255032787-158e76ba-834d-405a-9727-08aa81b60892.png "Portfolio Video")](https://youtu.be/sr2ndSHVX7w)
  </a>
</p>

Location: Anywhere - Remote, on-site
Technologies:  I'll adapt to wherever you guys are using.
Portfolio Website: https://fredy.dev
Email: me@fredy.dev
Skills: JS/TS, 3D Blender, Figma, Google Workspace, Linux.

[1]: https://github.com/vercel/next.js/assets/45242501/158e76ba-834d-405a-9727-08aa81b60892
[2]: https://youtu.be/sr2ndSHVX7w

**Comment by SiriusA015** [comment]
**TypeScript/React/Next.js/Node.js/GraphQL/AWS Developer**

Location: Sweden
Remote: Yes
Willing to relocate: No
Technologies: JavaScript/TypeScript, React/Next.js, Vue2/Vue3, Angular, NodeJS, PHP, MongoDB, AWS, Docker, MySQL, PostgreSQL
Resume/CV: https://drive.google.com/file/d/11txVJWSgrAQTQYTIpQCIbvZUf58n-BTF/view?usp=sharing
Website: https://[bio.link/pingstar](https://bio.link/pingstar)
Email: sirius.a015@gmail.com

**Comment by vr1eva** [comment]
Location: Peru
Remote: Yes
Willing to relocate: Yes
Technologies:I have over 6 years working with React and Next.js
Resume/CV: https://www.linkedin.com/in/bruno-gonzales-839878172
Website: vr1eva.netlify.app
Email: You can contact me sending an email to hivr1eva@gmail.com

**Comment by egzidev** [comment]
Location: Kosovo
Remote: Yes
Willing to relocate: No
Technologies: Typescript, NextJs, React, Figma, Tailwind, React Native, Docker, Node, React Query, Redux Toolkit
Resume/CV: https://www.linkedin.com/in/egzongollopeni/
Website: -
Email: [egzon.gollopeni@gmail.com](egzon.gollopeni@gmail.com)

**Comment by rhat4111** [comment]
**Senior Frontend Developer (React.js/Vue.js/Angular/Svelte/TailwindCSS)**

Location: Ukraine
Remote: Yes
Willing to relocate: No
Technologies: Senior Frontend Developer
Resume/CV: www.linkedin.com/in/eduard-moshkola-3641a5275
Website: https://eduard-moshkola.vercel.app

**Comment by LorenzoBloedow** [comment]
**Front-end Developer**
I code functional user interfaces and implement awesome designs :)

Location: Brazil
Remote: Yes
Willing to relocate: No
Technologies: TypeScript, Next.js, React, Redux, Tailwind CSS
Resume/CV: https://drive.google.com/file/d/1egKsimTl7Z1sboQlbA1SQ-yJuVUSnulG/view?usp=sharing
Website: https://lorenzobloedow.com
Email: lorenzobloedow@gmail.com

**Comment by LrLamonier** [comment]
Full-stack developer, Next.js + MERN

Hey! I'm Lucas and I have experience designing, developing, shipping, and maintaining solutions that reached thousands of users, helping them to bring their businesses to the internet and better connect with their customers.

Location: `Goiás, Brazil`
Remote: `yes, preferably`
Willing to relocate: `yes`
Technologies: `Next.js, MERN stack (Mongo, Express, React, Node), TailwindCSS, GSAP animation, and more`
Resume/CV: [resume in PDF](https://github.com/LrLamonier/LrLamonier/blob/main/lamonier-lucas-resume.pdf) 
Website: [LucasLamonier.com](https://www.lucaslamonier.com/) 
Email: [lucasrlamonier@gmail.com](mailto:lucasrlamonier@gmail.com)

**Comment by surnr** [comment]
### Hi, This is Surendar

Location: India
Remote: Yes
Willing to relocate: Yes
Technologies: NextJs, ReactJs, NodeJs, MongoDB, SQL, Redis, Docker, Kafka, RabbitMQ
Resume: https://drive.google.com/file/d/1cL_opcjmK2VsFky9iMGsZz1ZwVoCisMv/view
Website: https://www.linkedin.com/in/surendarsingh
Email: surendarkunn@gmail.com

**Comment by ahsanakhtar91** [comment]
Hi, my name is Ahsan. 
-----

I am a well-experienced **Front End Developer**, now having accumulatively more than **9** years of professional experience.

**Location**: Islamabad, Pakistan
**Remote**: Yes
**Willing to relocate**: Yes
**Role**: Sr. Front End Developer (ReactJS | React Native) - Web and/or Mobile

**Technologies**: React, React Native, JavaScript, TypeScript, Redux, React Native CLI, Expo, NextJS, Typescript, GraphQL, Tailwind CSS, Headless UI, React Hook Form, (and more Front End related technologies...).

**Resume/CV**: https://bit.ly/3VaLOnn
**LinkedIn**: https://www.linkedin.com/in/m-ahsan-akhtar-69772067
**Github**: https://github.com/ahsanakhtar91

I am immediately available and looking to take up a new role ASAP. You can contact me on my Email and Phone provided below:

**Email**: ahsanakhtar91@yahoo.com 📧
**Phone** +92 346 6817870 (WhatsApp) 📞

**Comment by maxiCalderonBuono** [comment]
## Hi!, I´m Max

Location: Copenhagen, Denmark
Remote: Yes
Willing to relocate: Yes
Role: Frontend Developer
Technologies: HTML, CSS, Javascript, React, NextJS, Typescript, Node, Express, MongoDB, Firebase, Supabase, Tailwind, Strapi, Contentful,
Resume/CV: https://drive.google.com/file/d/14CujtjpJGQg_glj2098YWtelH0SSwd-K/view?usp=drive_link
Website: https://www.maxicalderon.dev/
LinkedIn: https://www.linkedin.com/in/mcalderonbuono/
Github: https://github.com/maxiCalderonBuono

Open to Full-Time / Part-Time / Contract

**Comment by hugorydel** [comment]
Location: United States
Remote: Yes
Willing to relocate: No
Technologies: React.JS, Next.JS, Firebase, Postman, Node / Express, MongoDB, Figma, Bootstrap
Resume/CV: https://drive.google.com/file/d/1AYwag-HKlz-gmkn__KliNjL9kqeHmwwp/view?usp=sharing
Website: https://www.hugorydel.com/
Email: hugosteczkowski@gmail.com

A few example projects:
![Portfolio 2023 Cover Image v2](https://github.com/vercel/next.js/assets/56100537/2e16d57e-5818-4b86-ba05-087ae0ffec3a)
![Polymastro Cover](https://github.com/vercel/next.js/assets/56100537/b05f56ad-c517-457b-829c-56721eb27551)

**Comment by Dev-Ahmadubah** [comment]
Location: Nigeria
Remote: Yes
Willing to relocate: Maybe
Technologies: : HTML, CSS, JavaScript, Reactjs, Next.js, React Redux, Tailwind CSS...
Resume/CV: https://drive.google.com/file/d/1lMqH-ob7debxnC7J58zyUvd5Thx6nnf_/view
Website: https://ahmadubah.netlify.app/
Email: ahmadsta7@gmail.com

**Comment by DalareR** [comment]
Location: Columbia, PA
Remote: Yes
Willing to relocate: No
Technologies: React, NextJS, JavaScript, Typescript, Git, GitHub, Tailwind, Bootstrap, Chakra-UI, Styled Components  HTML5 & CSS3
Resume/CV: https://www.dropbox.com/scl/fi/9w3hg7q2ts1ar4igi1bxa/Dalare_Rana_Resume.pdf?rlkey=pxmgkdebeehw8f0tev0a9cyb1&dl=0
Website: https://dalarerana.com
Email: dalarerana@gmail.com

**Comment by kvnfo** [comment]
Hey all! My Name is Kevin

Location: Dallas, TX
Remote: Yes
Willing to relocate: No
Role: Frontend Developer
Technologies: React, NextJS, Tailwind, Typescript, Firebase, Supabase, Strapi, Contentful, Sanity
Website: https://www.phrame.studio
Github: https://github.com/kvnfo
Portfolio: https://phrame.studio

email: nivek.sayz@gmail.com

Open to Full-Time / Part-Time / Contract

**Comment by SuperButterfly** [comment]
Hi! My Name is Dragan Stevanovic

Location: Belgrade, Serbia
Remote: Yes
Willing to relocate: No
Role: Frontend Developer
Technologies: React,Symfony, Angular,NextJS, Tailwind, Typescript, Firebase

Github: https://github.com/superbutterfly

Gmail: topworldstar0101@gmail.com
Linkedin: https://www.linkedin.com/in/dragan-stevanovic-0b886a264/
Open to Full-Time / Part-Time / Contract

**Comment by kamaalsultan** [comment]
Hi! 
Location: Jennings, LA, US
Remote: Yes
Willing to relocate: No
Role: Frontend Developer & Backend Developer & Full Stack Developer
Technologies: Next.js, Nuxt.js, Angular, Supabase & Firebase, Tinybird, Stripe, Node.js, MySQL, MongoDB, Django Stack, AI Solution Development, Chatbot Development
Projects: Next.js + ChatGPT + Bard + Claude + Duckduckgo, Nuxt.js + Stripe + Tinybird + Supabase


Github: https://github.com/byteballet
Gmail: [caleb.rono1010@gmail.com](mailto:caleb.rono1010@gmail.com)

**Comment by juandavidGF** [comment]
**I build and deploy serverless AI base platforms**

Location: Colombia
Remote: Yes
Willing to relocate: No
Technologies: Nextjs, typescript, python, HugginFace, LLMs, Replicate, Langchain, SD, openai, aws sageMaker.
Resume/CV: https://www.linkedin.com/in/juandavidgf/
Website: juand4bot.com
Email: hi@juand4bot.com

Open to: Part time / freelance.

**Comment by rajibola** [comment]
Location: Nigeria
Remote: Yes
Willing to relocate: Yes
Website: https://folio-rajibola.vercel.app/  (Still in the oven)
Email: [rajibola1997@gmail.com](mailto:rajibola1997@gmail.com)

Hey, I'm Ridwan Ajibola, a frontend developer with over 5 years of experience. Check out what I've cooked up in my portfolio: https://folio-rajibola.vercel.app/ (Still in the oven)

I whip up Web Apps with ReactJs, NextJs, JavaScript, Typescript, and more.

I also dabble in Mobile App Crafting: React Native, Expo, React Navigation, and the gang.

On the hunt for a frontend developer role. Any help would make my day! Thanks a bunch.

**Comment by eybel** [comment]
Location: Rosario, Argentina & Barcelona, Spain
Remote: Yes
Willing to relocate: In Spain
Technologies: MERN / Next.js / TailwindCSS / AWS / Wordpress / others 
Resume/CV: https://drive.google.com/file/d/1neyDDhbHo4l6e9pdetuSHVb05mUOkIJ2/view?usp=sharing
Website: www.egubox.com
Email: abelmurua87@gmail.com 
Languages: Spanish / English

Good luck with interviewing!

**Comment by tamatamvan** [comment]
Location: Jakarta, Indonesia
Remote: Yes (any timezone)
Willing to relocate: Yes
Role: Fullstack Developer (Frontend Heavy)
Technologies: HTML, CSS, Javascript, TypeScript, React, NextJS, Node.js, Express, Nest.js, PostgreSQL, MongoDB, Firebase, Supabase, Tailwind, Laravel, Ruby on Rails, T3 stack, AWS, GCP, Docker, k8s. (Also willing to explore learn any tech stack not mentioned here.)
Website: https://septiantama.id/
CV/Resume: https://septiantama.id/resume
LinkedIn: https://www.linkedin.com/in/septian-tama/
Github: https://github.com/tamatamvan
Email: me@septiantama.id

Hi There, I'm Tama! I'm an experienced fullstack engineer with main expertise in frontend development.
I'm looking for new opportunities and open to full-time / part-time / contract positions.
If you have any opportunities that might be a good fit for me, feel reach me out through my contact above.
Any helps really appreciated, thanks a lot!

**Comment by Ree-m** [comment]
Location: Asia (GMT+3)
Remote: Yes, only remote (any time zone)
Willing to relocate: No
Technologies: JavaScript, React, Next, Node, Express, MongoDB, Puppeteer, Git/GitHub, REST API, Microservices, Deployment
Résumé/CV: https://bit.ly/3OlFXdD
Email:reembsrat@gmail.com
Website: https://reembsrat.com/
LinkedIn: https://www.linkedin.com/in/reembsrat/

Hi ! I’m Reem, a full-stack software developer. I have experience in both Frontend and Backend development. I take pride in being a fast learner and committed to continuous learning and expanding my skill set.

I have experience working with a team of remote developers. I am open to full-time ,part-time or contract positions.

**Comment by bravo-stack** [comment]
LOCATION : Nigeria

REMOTE: Yes

WILLING TO RELOCATE: Yes

TECHNOLOGIES: Typescript, ReactJS, NextJS, Redux, Styled-Components, SCSS, Redux-Toolkit, RTK Query, TailwindCSS

RESUME/CV: https://drive.google.com/file/d/1XuWGWQxH2TbWS6xCQN5xGho1j-uHIRoi/view

WEBSITE: https://bravostack.netlify.app/

LINKEDIN PROFILE: https://www.linkedin.com/in/chukwuebuka-onuh-625ba5227/

EMAIL: chukwuebukaonuh23@gmail.com

**Comment by odemur** [comment]
Developing software projects, unique experiences, and happy clients since 2001.

Location: São Paulo, Brazil 
Remote: Yes 
Willing to relocate: Yes, if required. 
Technologies: JavaScript, Typescript, MERN Stack, React, Node.js, Next.js, Tailwind CSS, CSS3, HTML5, AWS, Vercel

Resume/CV: https://www.odemur.com.br/software-engineer/resume-odemur-marangoni.pdf
Linkedin: https://www.linkedin.com/in/odemur
Website: https://www.odemur.com.br 
Email: dev@odemur.com.br

**Comment by Ella-1** [comment]
Location: Kaduna state, Nigeria
Remote: yes
Willing to relocate: No
Technologies: JavaScript/TypeScript, Next.JS, React, Node js, Tailwind CSS, Data Structures, React Query, Python, Docker, and many more
Resume/CV: https://twitter.com/Prof_Bata
Website: https://portfolio-prep.vercel.app/
Email: batazoaka3@gmail.com
